03-task-manager/starter/quizAnswers.txt

```diff
@@ -0,0 +1,4 @@
+1. Middleware is a function which we can evoke before we get the results for an API operation.
+2. Create, Read, Update, and Delete
```

Comment:
Very good answer. There are properties or features of middleware and it is always to a have more elaborate answers. It will come very useful to job interview process. You will learn more as you practice more about the corners and twists of middleware. Very important topic.  Great Job Suheyla.